test-poc/base/src/test/java/org/keycloak/test/base/ManagedResourcesTest.java

```diff
@@ -8,13 +8,14 @@
 import org.keycloak.test.framework.KeycloakIntegrationTest;
 import org.keycloak.test.framework.TestClient;
 import org.keycloak.test.framework.TestRealm;
+import org.keycloak.test.framework.injection.LifeCycle;
 
 import java.util.List;
 
 @KeycloakIntegrationTest
 public class ManagedResourcesTest {
 
-    @TestRealm
+    @TestRealm(lifecycle = LifeCycle.CLASS)
```

Comment:
The default `lifescycle` is `CLASS` so perhaps this can be omitted?
```suggestion
    @TestRealm
```

Reply:
Left it in there if someone wants to play around with changing it, which is what I did when implementing this ;)